sql-statements/sql-statement-load-stats.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Please add this file to TOC.md and fix lint. @qw4990

Reply:
Updated. PTAL.